<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomationsPage</class>
 <widget class="QWidget" name="AutomationsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="AutomationsSideBar" name="automationsSideBar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
     <widget class="QStackedWidget" name="automationsPageContainer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="AutomationsViewPage" name="automationsViewPage"/>
      <widget class="TrainingsViewPage" name="trainingsViewPage"/>
      <widget class="AutomationCreatePage" name="automationCreatePage"/>
      <widget class="AutomationDetailPage" name="automationDetailPage"/>
      <widget class="TrainingCreatePage" name="trainingCreatePage"/>
      <widget class="TrainingDetailPage" name="trainingDetailPage"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AutomationsSideBar</class>
   <extends>QWidget</extends>
   <header>automationssidebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AutomationsViewPage</class>
   <extends>QWidget</extends>
   <header>automationsviewpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AutomationDetailPage</class>
   <extends>QWidget</extends>
   <header>automationdetailpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AutomationCreatePage</class>
   <extends>QWidget</extends>
   <header>automationcreatepage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TrainingsViewPage</class>
   <extends>QWidget</extends>
   <header>trainingsviewpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TrainingDetailPage</class>
   <extends>QWidget</extends>
   <header>trainingdetailpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TrainingCreatePage</class>
   <extends>QWidget</extends>
   <header>trainingcreatepage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
